wfoverlap/default.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv, lib, fetchFromGitHub, gfortran, makeWrapper, blas-i8, lapack }:
+assert
+  lib.asserts.assertMsg
+  (blas-i8.isILP64 || blas-i8.passthru.implementation == "mkl")
+  "64 bit integer BLAS implementation required.";
+
+let
+  useMKL64 = blas-i8.passthru.implementation == "mkl";
```

Comment:
I gave it a quick try and removed all of MKL distinction and could build it without problem against MKL (`nix-build release.nix -A qchem-mkl.wfoverlap --arg buildVariants true`). I think we can remove all of that code?